projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html

```diff
@@ -157,7 +157,7 @@
         </div>
 
         <!-- BUTTON SECTION -->
-        <div class="cx-payment-form__btns row">
+        <div class="row">
```

Comment:
Shouldn't this be `cx-btns`? In mobile the two buttons stack without any margin 😱

Reply:
Or is that part of the follow up

Reply:
I'll take note of this and fix these kind of issues in ticket 1241

Reply:
👍